test/integration_test/specs/mysql-1-pvc-pre-exec-rule-backup/mysql-pre-backup-rule.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: Rule
+metadata:
+  name: mysql-pre-backup-rule
+  namespace: mysql-1-pvc-mysql
+  annotations:
+    "stork/cmdexecutor-image": "openstorage/cmdexecutor:latest"
+rules:
+  - podSelector:
+      app: mysql
+    actions:
+    - type: command
+      # dummy action to test non-background command
+      value: echo "hello stork pre-backup"
+      runInSinglePod: true
+    - type: command
+      runInSinglePod: true
```

Comment:
`background` needs to be set to `true` for this and the next command

Reply:
Fixed.